OrbitCore/CoreApp.h

```diff
@@ -34,6 +34,8 @@ class CoreApp {
   virtual void InitializeClientTransactions();
   virtual void SendToUiAsync(const std::string& /*message*/) {}
   virtual void SendToUiNow(const std::string& /*message*/) {}
+  virtual void SendErrorToUi(const std::string& /*title*/,
+                             const std::string& /*text*/) {}
```

Comment:
I do not think it is needed here - looks like you call it only from Gl and can use OrbitApp instead of CoreApp.

Reply:
Done.